.circleci/config.yml

```diff
@@ -1182,12 +1182,18 @@ jobs:
       - revenuecat/install-maestro
       - tuist-generate-workspace:
           query: "Maestro"
+      - run:
+          name: Set API key for e2e tests
+          command: |
+            if [ "<< parameters.backend_environment >>" == "production_test_store" ]; then
+              echo 'export REVENUECAT_API_KEY="${RC_E2E_TEST_API_KEY_PRODUCTION_TEST_STORE}"' >> $BASH_ENV
+            else
+              echo 'export REVENUECAT_API_KEY="${RC_E2E_TEST_API_KEY_PRODUCTION_APP_STORE}"' >> $BASH_ENV
+            fi
       - run:
           name: Run e2e tests (<< parameters.backend_environment >>)
           command: bundle exec fastlane run_maestro_e2e_tests backend_environment:<< parameters.backend_environment >>
           no_output_timeout: 15m
-          environment:
-            REVENUECAT_API_KEY: ${RC_E2E_TEST_API_KEY_PRODUCTION_APP_STORE}
 
         # todo: implement error / success notifications through Slack
```

Comment:
Just noticed this, this was implemented in the meantime. Could you remove this line in this PR? Alternatively I can spin up a second one.